src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs

```diff
@@ -20,6 +20,23 @@ public partial class BinaryFormatterTests
         // *** AUTO UPDATED BLOBS ***
         private static DateTime s_fixedTimestamp = DateTime.Parse("11/3/1989 04:50:29 AM", CultureInfo.InvariantCulture.DateTimeFormat);
 
+        public static IEnumerable<object[]> SerializableObjects_MemberData()
+        {
+            // Save old culture and set a fixed culture for object instantiation
+            CultureInfo oldCulture = CultureInfo.CurrentCulture;
+
+            try
+            {
+                CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
```

Comment:
Is there any way to refactor and run this code in RemoteInvoke? setting the current culture like that can potentially cause some problems for other tests.

Reply:
I know you are porting the changes. I just talking in general refactoring the tests on master